tests/vmi_configuration_test.go

```diff
@@ -346,6 +346,35 @@ var _ = Describe("Configurations", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 		})
+
+		Context("with guestAgent", func() {
+			var agentVMI *v1.VirtualMachineInstance
+
+			BeforeEach(func() {
+				agentVMI = tests.NewRandomVMIWithEphemeralDisk(tests.RegistryDiskFor(tests.RegistryDiskAlpine))
+				By("Starting a VirtualMachineInstance")
+				agentVMI, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(agentVMI)
+				Expect(err).ToNot(HaveOccurred(), "Should create VMI successfully")
+				tests.WaitForSuccessfulVMIStart(agentVMI)
+			})
+
+			It("there should be agent channel by default", func() {
```

Comment:
We will need an image with a guest agent installed. We could probably build and customize the fedora cloud image which we include as a registry disk already in the `kubevirtci` repo. Important is that we can not assume internet connectivity during the test run itself. Do you have other ideas?

Reply:
Could we also use the stock fedora image, then install the guest agent using cloud-init?

Reply:
We could, but then our tests can't be run offline anymore.

Reply:
Tests should not assume anything. We require image with GA, therefore we provide it.
Fedora with installed GA is fine. I would do it the same way as we do other images. Put it in the registry disk.